### ISSUE TITLE
App doesn't run on Mac

### ISSUE THREAD

**Issue by paulrosen** [Jermolene/TiddlyDesktop#114]
MacbookPro running El Capitan.

I downloaded the file and unzipped it. When I double click it, a dialog pops up that says "verifying TiddlyWiki.app" and that dialog just sits there.

**Comment by krvs-esoptra** [comment]
+1

**Comment by rvs-fluid-it** [comment]
+1

**Comment by Jermolene** [comment]
Hi @paulrosen @rvs-fluid-it thanks for the report. I've tested the app on Mac OS Sierra, but don't have access to a copy of El Capitan. I haven't been able to find any reference to the nw.js platform not being able to run on El Capitan.

It may be worth running the app from the command line so that you can see any error messages that are displayed. To do so, open a terminal and type `/Applications/TiddlyWiki.app/Contents/MacOS/nwjs` (assuming your copy is stored in the root "Applications" folder).

**Comment by rvs-fluid-it** [comment]
Running the App from the command line is possible. Thanks for the workaround.

**Comment by Jermolene** [comment]
Have you tried right-clicking on the app icon and choosing "Open"? If the problem is Gatekeeper that should give you a dialogue allowing you to approve running the app...

**Comment by rvs-fluid-it** [comment]
Yes, I did. But I will give it another try.  It seems that it takes a long time before the Gatekeeper related is opened.

**Comment by Jermolene** [comment]
Thanks @rvs-fluid-it

**Comment by jbl0ndie** [comment]
That reminds me of a similar issue I get with Libre Office which I think relates to Gatekeeper. The issue and a workaround using `xattr` from the command line is described here.

https://ask.libreoffice.org/en/question/64253/libreoffice-513-stuck-on-verifying-mac-os-x-yosemite-10105/
